notes/release.md

```diff
@@ -1,51 +1,52 @@
-# Release (WIP)
+# Release
 
 This note explains how to release deepchem packages.
 
 ## How to release
 
-1. Create and merge a release PR (just modify the version in `deepchem/__init__.py`)
-2. Push a new tag in the merge commit -> release in PyPI
-3. Create and merge a release PR in the [feedstock repository](https://github.com/conda-forge/deepchem-feedstock) -> release in conda forge
-4. Create and merge a PR for updating the Dockerfile (`docker/conda-forge/Dockerfile`)
-5. Build and publish a new docker image -> release in DockerHub 
-6. Fix the version in README.md and Documentation
+1. Create and merge a release PR
+    - Modify the version in `deepchem/__init__.py` (Remove `.dev`)
+    - Update the documents for installing a new package in `README.md` and `docs`
+2. Push a new tag to the merge commit -> release new PyPI package and docker image
+3. Create and merge a release PR in the [feedstock repository](https://github.com/conda-forge/deepchem-feedstock) -> release new Conda Forge package
+4. Publish the documents for a new tag in [ReadTheDocs](https://readthedocs.org/projects/deepchem/versions/).
```

Comment:
Additional process.

We should publish the docs for an each tag because of backward compatibility

Reply:
Good addition here!